trailblazer/server/api.py

```diff
@@ -76,5 +78,8 @@ def me():
 @blueprint.route('/aggregate/jobs')
 def aggregate_jobs():
```

Comment:
we could even give it a param that defaults to 31. Then we can surf to `/aggregate/jobs/60` and see more ;)

I don't know where this is used, so I don't know if this makes sense.

Reply:
I think there is more to it, it's not the address of the backend that we use in the browser so we need to relay what is written there through the Vue application